pioneer.json

```diff
@@ -0,0 +1,25 @@
+{
+  "response": [
+    {
+        "maintainer": "Ray (K!lly)",
+        "oem": "Sony",
+        "device": "Xperia XA2",
+        "filename": "crDroidAndroid-10.0-20191206-pioneer-v6.0.zip",
+        "download": "https://sourceforge.net/projects/crdroid/files/pioneer/6.x/crDroidAndroid-10.0-20191206-pioneer-v6.0.zip/download",
+        "timestamp": 1575572437,
+        "md5": "5d9870baddc6883f33821b8ec4d0c249",
+        "size": 626122638,
+        "version": "6.0",
+        "buildtype": "Beta",
+        "forum": "https://forum.xda-developers.com/xperia-xa2/development/rom-crdroidandroid-v6-0-t4015327",
+        "forum": "http://4pda.ru/forum/index.php?s=&showtopic=908380&view=findpost&p=91600450",  
```

Comment:
you can only have one key named forum
please change file and update your pull request

Reply:
I've no changes bcoz I follow for LOS. And this 1st publish of release

Reply:
"forum":
did it, only XDA